www/src/pages/pt/deployment/docker.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Docker
+description: Deploy com Docker
+layout: ../../../layouts/docs.astro
+lang: pt
+---
+
+Você pode colocar essa stack em contêineres e fazer deploy dela como um único contêiner usando o Docker ou como parte de um grupo de contêineres usando o docker-compose. Veja [`ajcwebdev/ct3a-docker`](https://github.com/ajcwebdev/ct3a-docker) para um repositório de exemplo baseado neste documento.
+
+## Configuração do Projeto Docker
+
+Por favor, note que o Next.js requer um processo diferente para tempo de build (disponível no frontend, prefixado por `NEXT_PUBLIC`) e ambiente de tempo de execução, somente do lado do servidor, variáveis. Nesta demonstração estamos usando duas variáveis, preste atenção em suas posições no `Dockerfile`, argumentos de linha de comando e `docker-compose.yml`:
+
+- `DATABASE_URL` (usado pelo servidor)
+- `NEXT_PUBLIC_CLIENTVAR` (usado pelo cliente)
+
+### 1. Próxima configuração
+
+Em seu [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), adicione a ` configuração de opção de saída autônoma para [reduzir o tamanho da imagem aproveitando automaticamente os rastreamentos de saída](https://nextjs.org/docs/advanced-features/output-file-tracing):
+
+```diff
+export default defineNextConfig({
+  reactStrictMode: true,
+  swcMinify: true,
++ output: "standalone",
+});
+```
+
+### 2. Criar arquivo dockerignore
+
+<details>
+    <summary>
+      Clique aqui e inclua esse conteúdos no <code>.dockerignore</code>:
+    </summary>
+<div class="content">
+
+```
+.env
+Dockerfile
+.dockerignore
+node_modules
+npm-debug.log
+README.md
+.next
+.git
+```
+
+</div>
+
+</details>
+
+### 3. Crie o Dockerfile
+
+> Como não estamos puxando as variáveis de ambiente do servidor para nosso contêiner, a [validação do esquema do ambiente](/en/usage/env-variables) falhará. Para evitar isso, temos que adicionar a flag `SKIP_ENV_VALIDATION=1` ao comando de compilação para que os schemas env não sejam validados no momento da compilação.
```

Comment:
```suggestion
> Como não estamos puxando as variáveis de ambiente do servidor para o nosso contêiner, a [validação do esquema do ambiente](/en/usage/env-variables) falhará. Para evitar isso, temos que adicionar a flag `SKIP_ENV_VALIDATION=1` ao comando de build para que os esquemas env não sejam validados no momento de build.
```

I changed in the suggestion `compilação` to `build`, because I thought in other places even in Portuguese in this PR, the word "build" is kept as is.
But I am not sure which would be better.